src/styles/_loading.scss

```diff
@@ -0,0 +1,41 @@
+#loading-container {
+  position: relative;
+  height: 100%;
+  width: 100%;
+  background-color: rgba(0, 0, 0, 0.4);
+  z-index: 600;
+}
+
+#loader {
+  position: absolute;
+  left: 50%;
+  top: 50%;
+  margin: -75px 0 0 -75px;
+  border: 16px solid #f3f3f3;
+  border-radius: 50%;
+  border-top: 16px solid $rent-strike-color;
+  width: 100px;
+  height: 100px;
+  -webkit-animation: spin 1s linear infinite;
+  animation: spin 1s linear infinite;
+}
+
+@-webkit-keyframes spin {
```

Comment:
I checked the CSS animation on IE11 & Firefox, and it looks good. Haven't yet checked mobile, Safari, or Chrome.

Reply:
Nice.